tracker/tracker_debugger_api.yaml

```diff
@@ -61,6 +61,41 @@ paths:
             example: john@doe.com
           in: query
           required: false
+        - name: page_url
+          description: Filter. A part of page URL of the event, that have to be in the session to display the session
+          schema:
+            type: string
+            example: blog
+          in: query
+          required: false
+        - name: event_type
+          description: Filter. A type of the event, that have to be in the session to display the session
+          schema:
+            type: int
+            format: uint8
+            example: 1
+            enum:
+              - 1 - page view
```

Comment:
Enum values should be integers too. The description is a comment for us.
```suggestion
              - 1 # - page view
```

Reply:
Yes, currently enum value is presented as a number with label pair. Enum should contain only numbers and explanation for them may be added in description if it's necessary.
![2021-06-29T134934](https://user-images.githubusercontent.com/4488774/123792827-5a59ae00-d8e1-11eb-9f9e-811d505fcf22.png)

Reply:
It renders like this:
![image](https://user-images.githubusercontent.com/12272711/123793746-5c703c80-d8e2-11eb-87fa-d4e999bec18f.png)

I'm not conviced that user should know only of possible values, since this is a public API. I think that if somebody would like to use this endpoint he should not look for the meaning of the values in other places or try to figure out those meanings from our UI.

Reply:
Ok, i will try with the description.